### DISCUSSION TITLE
MemGPT like memory plugin

### DISCUSSION THREAD

**Discussion by AshD** [microsoft/semantic-kernel#4770] [Q&A] [answered]
Is there a MemGPT like plugin that **automatically** records interactions that integrates with Semantic Kernel?
https://memgpt.readme.io/docs/index

Thanks,
Ash

**Comment by sophialagerkranspandey** [comment, marked as answer]
Today we don't have that. As a part of Build we are revisiting our memory abstractions and will see if we can include this functionality.

**Reply by tyler-suard-parker** [reply]
It's 2 months later, is this feature implemented?
